CONTRIBUTING.md

```diff
@@ -49,3 +49,20 @@ In addition to contributing code, you can help to triage issues. This can includ
 reports or asking for vital information, such as version numbers or reproduction instructions. If
 you would like to start triaging issues, one easy way to get started is to [subscribe to backupii on
 CodeTriage](https://www.codetriage.com/backupii/backupii).
+
+## Getting started
+
+BackupII is a Ruby app that is unit and integration tested with Rspec and linted with Rubocop:
+
+```bash
+bundle
+bundle exec rubocop
+bundle exec rspec spec
+bundle exec rspec integration
+```
+
+There is also a [Dockerfile](Dockerfile) and respective Rake tasks to run all the tests in Docker:
+
+```bash
+bundle exec rake docker:prepare docker:spec docker:integration
+```
```

Comment:
I'm not sure the Docker thing was completely finished and supported by the previous maintainer(s), especially the integration part. I assume it works well or you wouldn't have submitted the PR :)
Can you confirm this or open an issue if you find something is lacking ?

Reply:
Mhh, I hadn't had any issues with any test failing. The same amount of tests run inside and outside of Docker. So I think everything is working fine.

Reply:
Awesome. Thanks for the answer